src/role-access/service/index.ts

```diff
@@ -0,0 +1,5 @@
+import { RoleExclusionsService } from './role-exclusions.service';
+
+export const services: any[] = [RoleExclusionsService];
+
+export * from './role-exclusions.service';
```

Comment:
would this file move to src/role-access/services but not src/role-access/service

Reply:
Done